pkg/provider/aws/edges/edges.go

```diff
@@ -0,0 +1,117 @@
+package edges
```

Comment:
To go along with a previous PR's comment regarding a `pkg/knowledgebase` package, I'd prefer this in an `aws/knowledgebase` package. We have a lot of different edges so we should be wary of potentially ambiguous names especially when it comes to packages (compared to types, which are always package-prefixed).

Reply:
changed the package name